recipes/sawfish/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+# Sawfish already auto-extracts to bin directory, so just need to move it to PREFIX 
+cp -r . "${PREFIX}"/
```

Comment:
I am not sure this is good.

```
tar xvf sawfish-v0.12.1-x86_64-unknown-linux-gnu.tar.gz 
sawfish-v0.12.1-x86_64-unknown-linux-gnu/
sawfish-v0.12.1-x86_64-unknown-linux-gnu/bin/
sawfish-v0.12.1-x86_64-unknown-linux-gnu/bin/sawfish
sawfish-v0.12.1-x86_64-unknown-linux-gnu/data/
sawfish-v0.12.1-x86_64-unknown-linux-gnu/data/expected_cn/
sawfish-v0.12.1-x86_64-unknown-linux-gnu/data/expected_cn/expected_cn.hg19.XX.bed
sawfish-v0.12.1-x86_64-unknown-linux-gnu/data/expected_cn/expected_cn.hg19.XY.bed
sawfish-v0.12.1-x86_64-unknown-linux-gnu/data/expected_cn/expected_cn.hg38.XX.bed
sawfish-v0.12.1-x86_64-unknown-linux-gnu/data/expected_cn/expected_cn.hg38.XY.bed
sawfish-v0.12.1-x86_64-unknown-linux-gnu/data/expected_cn/expected_cn.hs37d5.XX.bed
sawfish-v0.12.1-x86_64-unknown-linux-gnu/data/expected_cn/expected_cn.hs37d5.XY.bed
sawfish-v0.12.1-x86_64-unknown-linux-gnu/data/README.md
sawfish-v0.12.1-x86_64-unknown-linux-gnu/docs/
sawfish-v0.12.1-x86_64-unknown-linux-gnu/docs/accuracy.md
sawfish-v0.12.1-x86_64-unknown-linux-gnu/docs/user_guide.md
sawfish-v0.12.1-x86_64-unknown-linux-gnu/img/
sawfish-v0.12.1-x86_64-unknown-linux-gnu/img/logo.svg
sawfish-v0.12.1-x86_64-unknown-linux-gnu/CHANGELOG.md
sawfish-v0.12.1-x86_64-unknown-linux-gnu/LICENSE.md
sawfish-v0.12.1-x86_64-unknown-linux-gnu/README.md
```

So, this content will be extracted in the Conda environment. The README.md, LICENSE.md and CHANGELOG.md will be at the "top" of the environment.
Usually software should install such files into its own sub-folder, e.g. $PREFIX/share/sawfish/...
The binary should be in $PREFIX/bin because this folder is added to the $PATH, but anything else should be structured.

Reply:
Thanks for clarifying. I simplified the install to add only `bin/sawfish` now.